README.md

```diff
@@ -18,11 +18,11 @@ implementation("com.spotify.confidence:openfeature-provider-android:0.1.7")
 ```
 It can also be consumed from jitpack for using any branch or build:
 ```
-implementation("com.github.spotify:confidence-openfeature-provider-kotlin:[BRANCH]-[SNAPSHOT/Version]")
+implementation("com.github.spotify:confidence-sdk-android:[BRANCH]-[SNAPSHOT/Version]")
 ```
 for using specific commit:
 ```
-implementation("com.github.spotify:confidence-openfeature-provider-kotlin:[COMMIT SHA]")
+implementation("com.github.spotify:confidence-sdk-android:[COMMIT SHA]")
 ```
```

Comment:
does this still work? and how does it work with a multi module repo?
Maybe better do ditch this completely.